test/step-definitions/consent-requests-validate-outbound.step.ts

```diff
@@ -0,0 +1,171 @@
+/*****
+ License
+ --------------
+ Copyright © 2020 Mojaloop Foundation
+ The Mojaloop files are made available by the Mojaloop Foundation under the Apache License, Version 2.0 (the "License")
+ and you may not use these files except in compliance with the License. You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, the Mojaloop files are distributed
+ on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and limitations under the License.
+ Contributors
+ --------------
+ This is the official list of the Mojaloop project contributors for this file.
+ Names of the original copyright holders (individuals or organizations)
+ should be listed with a '*' in the first column. People who have
+ contributed from an organization can be listed under the organization
+ that actually holds the copyright for their contributions (see the
+ Gates Foundation organization for an example). Those individuals should have
+ their names indented and be marked with a '-'. Email address can be added
+ optionally within square brackets <email>.
+
+  - Kevin Leyow <kevin.leyow@modusbox.com>
+
+ --------------
+ ******/
+
+import { Server, ServerInjectResponse } from '@hapi/hapi'
+import { ServerAPI, ServerConfig } from '~/server'
+import { defineFeature, loadFeature } from 'jest-cucumber'
+import Config from '~/shared/config'
+import Handlers from '~/handlers'
+import index from '~/index'
+import path from 'path'
+import { NotificationCallback, Message, PubSub } from '../../src/shared/pub-sub';
+import { RedisConnectionConfig } from '../../src/shared/redis-connection';
+import { thirdparty as tpAPI } from '@mojaloop/api-snippets';
+
+const apiPath = path.resolve(__dirname, '../../src/interface/api-outbound.yaml')
```

Comment:
```suggestion
const apiPath = path.resolve(__dirname, '~/interface/api-outbound.yaml')
```

Reply:
I don't think that works with path.resolve?

Reply:
yeah, you are right -> `path.resolve` knows nothing about this